media/js/base/fxa-product-button.es6.js

```diff
@@ -169,7 +169,7 @@ FxaProductButton.init = function (overwrite) {
     // Collect all Fxa product buttons
     _buttons = document.getElementsByClassName('js-fxa-product-button');
 
-    if (!FxaProductButton.isSupported() || _buttons.length === 0) {
```

Comment:
Note for reviewer: removed this here since it is already covered by a check on line 177 below.